boa3_test/tests/compiler_tests/test_metadata.py

```diff
@@ -197,6 +197,26 @@ def test_metadata_info_supported_standards_with_imported_event(self):
         self.assertIn('name', abi['events'][0])
         self.assertEqual('Transfer', abi['events'][0]['name'])
 
+    def test_metadata_info_supported_standards_filter(self):
+        path = self.get_contract_path('MetadataInfoSupportedStandardsFilter.py')
+        output, manifest = self.compile_and_save(path)
+
+        self.assertIn('supportedstandards', manifest)
+        self.assertIsInstance(manifest['supportedstandards'], list)
+        self.assertGreater(len(manifest['supportedstandards']), 0)
+        self.assertIn('NEP-17', manifest['supportedstandards'])
+
+        # Using a dot is permitted and won't be removed by the filter
+        self.assertIn('NEP-17.1', manifest['supportedstandards'])
```

Comment:
I think the regex only matches the case for one dot. Versioning usually uses 2 dots, so we'd need to support at least that.
Can you add some more tests related to this?